client/components/premium-global-styles-upgrade-modal/index.tsx

```diff
@@ -54,16 +73,21 @@ export default function PremiumGlobalStylesUpgradeModal( {
 								<>
 									<p>
 										{ translate(
-											"You've selected a custom style that will only be visible to visitors after upgrading to the Premium plan or higher."
+											"You've selected a custom style that will only be visible to visitors after upgrading to the Premium plan or higher.",
```

Comment:
This might be a good chance to update the apostrophes in the modals, there is one here in `You've` that should be `You’ve` and the other is in `It's` in the file [design-setup/upgrade-modal.tsx#L74](https://github.com/Automattic/wp-calypso/blob/3bf77a982049d47b3711aca5e65e24db6cf7bdd8/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/upgrade-modal.tsx#L74)

For reference: puPv3-6m9-p2#comment-19138

Reply:
Nice catch! Will update it later 👍